www/js/app.js

```diff
@@ -145,6 +145,32 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         document.getElementById('searchArticles').click();
         return false;
     });
+     //Handle Home key press to focus #prefix
+     $(window,document).on('keydown', function(e) {
+         //if home key is pressed then
+         if(/^Home/.test(e.key)) {
+             //scroll to top of #search-article section since scroll bar is associated with it and not to window or document
+             $('#search-article').scrollTop(0);
+             $('#prefix').focus();
+         }
+     });
+     //Handle Home key event when focus is inside iframe #articleContent
+     var iframe= document.getElementById('articleContent');
+     //onload event is not fired in chrome for iframe if content is from local storage
+     //to compensate for onload event setTimeout can be used
```

Comment:
We're already using the onload event of the iframe in `readArticle` function. And it seems to work well (even on Chrome).
Using `setInterval` here is unsafe : it might run too early or too late

Reply:
When I tried using onload event, the changes were not working as expected. Now I understand that it was not due to problem with firing of onload event. The problem was that onload event handler is redefined by `readArticle` function and `displayArticleContentInIframe` function whenever called.
Then the keydown event handler for iframe must be defined in these functions inside onload handler. 
Please do guide me if I am wrong.

Reply:
I have already implemented as described above. Only issue remains in case of initial empty home page (or initial empty article.html loaded in iframe). In this case iframe.onload does not get triggered.
A possible solution is to add keydown handler to iframe when the document is ready.